libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -690,4 +702,23 @@ export class ChangeAction extends BaseRushAction {
   private _logNoChangeFileRequired(): void {
     console.log('No changes were detected to relevant packages on this branch. Nothing to do.');
   }
+
+  private _stageGitChanges(patterns: string[]): void {
+    try {
+      patterns.forEach((pattern) => {
+        child_process.execSync(`git add "${pattern}"`);
+      });
+    } catch (error) {
+      console.error(`Cannot stage git changes ${(error as Error).message}`);
```

Comment:
is this the usual way of logging in rush? Seems to be, tho I assumed they'd use custom logger wrappers

Reply:
🤷‍♂️

Reply:
Use `this._terminal.writeErrorLine`